bitmovin-analytics-datasource/src/types.ts

```diff
@@ -21,6 +21,7 @@ export interface BitmovinAnalyticsDataQuery extends DataQuery {
   orderBy: QueryOrderBy[];
   limit?: number;
   filters: QueryFilter[];
+  aliasBy?: string;
```

Comment:
Why are you adding this field to the query?
According to our api there is no aliasBy on the query

Reply:
This is the DataQuery interface, not the RequestQuery Interface. So this interface defines the types that are possible to set in the QueryEditor and get passed to the query method, where the actual RequestQuery is then built.